addons/im_livechat/report/im_livechat_report_channel_views.xml

```diff
@@ -1,46 +1,71 @@
+            <field name="name">im_livechat.report.operator.search</field>
             <field name="model">im_livechat.report.channel</field>
             <field name="arch" type="xml">
-                <pivot string="Livechat Support Statistics" disable_linking="1" sample="1">
-                    <field name="technical_name" type="row"/>
-                    <field name="duration" type="measure" string="Duration of Session (min)"/>
-                    <field name="nbr_message" type="measure" string="Messages per session"/>
-                    <field name="is_without_answer" invisible="True"/>
-                </pivot>
+                <search string="Search report">
+                    <field name="partner_id"/>
+                    <filter name="last_24h" string="Last 24h" domain="[('start_date','&gt;', (context_today() - datetime.timedelta(days=1)).strftime('%Y-%m-%d') )]"/>
+                    <filter name="start_date_filter" string="This Week" domain="[
+                        ('start_date', '>=', (datetime.datetime.combine(context_today() + relativedelta(weeks=-1,days=1,weekday=0), datetime.time(0,0,0)).to_utc()).strftime('%Y-%m-%d %H:%M:%S')),
+                        ('start_date', '&lt;', (datetime.datetime.combine(context_today() + relativedelta(days=1,weekday=0), datetime.time(0,0,0)).to_utc()).strftime('%Y-%m-%d %H:%M:%S'))]"/>
+                    <filter name="last_month" string="This Month" domain="[
+                        ('start_date', '>=', (datetime.datetime.combine(context_today() + relativedelta(months=-1,days=1,weekday=0), datetime.time(0,0,0)).to_utc()).strftime('%Y-%m-%d %H:%M:%S')),
+                        ('start_date', '&lt;', (datetime.datetime.combine(context_today() + relativedelta(days=1,weekday=0), datetime.time(0,0,0)).to_utc()).strftime('%Y-%m-%d %H:%M:%S'))]"/>
+                    <separator/>
+                    <separator/>
+                    <filter name="filter_start_date" date="start_date"/>
+                    <group expand="0" string="Group By...">
+                        <filter name="group_by_channel" string="Channel" domain="[]" context="{'group_by':'channel_id'}"/>
+                        <filter name="group_by_operator" string="Operator" domain="[('partner_id','!=', False)]" context="{'group_by':'partner_id'}"/>
+                        <separator orientation="vertical" />
+                        <filter name="group_by_month" string="Creation date" domain="[]" context="{'group_by':'start_date:month'}" />
+                    </group>
+                </search>
             </field>
         </record>
 
-        <record id="im_livechat_report_channel_view_graph" model="ir.ui.view">
-            <field name="name">im_livechat.report.channel.graph</field>
+        <record id="im_livechat_report_operator_view_pivot" model="ir.ui.view">
+            <field name="name">im_livechat.report.operator.pivot</field>
```

Comment:
Same, this shouldn't be renamed, but be added at the top, with the existing im_livechat_report_channel_view_pivot